exp-player/addon/components/exp-player.js

```diff
@@ -34,51 +47,71 @@ export default Ember.Component.extend({
         }
         return componentName;
     }),
+    currentFrameId: Ember.computed('currentFrame', function() {
+        var currentFrame = this.get('currentFrame');
+        return currentFrame.id;
+    }),
+    currentFrameData: Ember.computed('currentFrame', function() {
+        var currentFrame = this.get('currentFrame');
+        var context = this.get('expData');
 
-    actions: {
-        saveFrame(frameId, frameData) {
-            // Save the data from a completed frame to the session data item
-            var expData = this.get('expData');
-            expData[frameId] = frameData;
-            this.set('expData', expData);
-        },
-        saveSession() {
-            // Construct payload and send to server
-            var frames = this.get('frames');
-            var sequence = frames.map((frame) => frame.id);
+        if (!context[currentFrame.id]) {
+            context[currentFrame.id] = null;
+        }
+        return context[currentFrame.id];
+    }),
+    currentFrameCtx: Ember.computed('currentFrame', function() {
```

Comment:
Note for reviewers: it seems that this (and quite a few other places in the code) is the result of a messy merge conflict. 

This code was removed in the exp-player refactor that landed with #19, and some cleanup will probably be necessary before merging this into develop to avoid regressions.